src/router/index.js

```diff
@@ -0,0 +1,15 @@
+import Vue from "vue";
```

Comment:
This file's job is to set up Vue Router, which allows us to define any number of "routes" which tells a particular component to appear at a particular URL / folder structure

Reply:
Yes, minus the bit about folder structure.

The router will display different components based on the URL in the browser.